app/app.go

```diff
@@ -721,7 +721,9 @@ func NewApp(
 	// CanWithdrawInvariant invariant.
 	// NOTE: staking module is required if HistoricalEntries param > 0
 	// NOTE: capability module's beginblocker must come before any modules using capabilities (e.g. IBC)
-	app.ModuleManager.SetOrderBeginBlockers(
+
+	// Bypass app.ModuleManager.SetOrderBeginBlockers as we need to override the missing `mint` module check
+	app.ModuleManager.OrderBeginBlockers = []string{
```

Comment:
@Reecepbcups, any idea why I need this fix? Previous versions were working just fine.

Reply:
SetOrderBeginBlockers calls assertNoForgottenModules.

Make sure the sdk fork has mint removed, does not seem you built off my branch.

Also you removed distr recently yea? distribution is in this list so assertNoForgottenModules prob sees its not set up and errors

Reply:
Fixed by https://github.com/liftedinit/cosmos-sdk/pull/1/commits/c8c96a0017c6f29ac34a8a764eecafafcd6b2830 and using https://github.com/liftedinit/cosmos-sdk/releases/tag/v0.50.8-liftedinit.1